org.libvips.nip4.json

```diff
@@ -0,0 +1,478 @@
+                    "tag" : "v3.3.1",
+                    "commit" : "f723c3940fff287c0a26b425b90a8e764823bfd4"
+                }
+            ]
+        },
+
+        {
+            "name" : "uthash",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "install -D include/* /app/include"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/troydhanson/uthash.git",
+                    "commit" : "ca98384ce7f30beb216f9a0bc88a3b4340ead729"
+                }
+            ]
+        },
+
+        {
+            "name" : "libdicom",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--buildtype=release",
+                "--libdir=lib",
+                "-Dtests=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/ImagingDataCommons/libdicom.git",
+                    "tag" : "v1.1.0",
+                    "commit" : "60f291dea0fd464e34d693b1ad55c46f9f2ee69a"
+                }
+            ]
+        },
+
+        {
+            "name" : "openslide",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--libdir=lib",
+                "--buildtype=release"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/openslide/openslide/releases/download/v4.0.0/openslide-4.0.0.tar.xz",
+                    "sha256" : "cc227c44316abb65fb28f1c967706eb7254f91dbfab31e9ae6a48db6cf4ae562"
+                }
+            ]
+        },
+
+        {
+            "name" : "cfitsio",
+            "builddir" : true,
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.5.0.tar.gz",
+                    "sha256" : "e4854fc3365c1462e493aa586bfaa2f3d0bb8c20b75a524955db64c27427ce09"
+                }
+            ]
+        },
+
+        {
+            "name" : "libde265",
+            "builddir" : true,
+            "buildsystem" : "cmake-ninja",
+            "build-options" : {
+                "config-opts" : [
+                    "-DCMAKE_INSTALL_LIBDIR=lib"
+                ]
+            },
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/strukturag/libde265/releases/download/v1.0.15/libde265-1.0.15.tar.gz",
+                    "sha256" : "00251986c29d34d3af7117ed05874950c875dd9292d016be29d3b3762666511d"
+                }
+            ]
+        },
+
+        {
+            "name" : "libheif",
```

Comment:
libheif 1.18.2 is in the runtime

Reply:
done, removed libde265 too

Reply:
de265 isn't in the runtime because lawyers.

Reply:
it's in the ffmpeg-full extension.